docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -94,10 +97,80 @@ Priority levels
+
+If P=1 becomes unhealthy, it will continue to take spilled load from P=0 until the sum of the health
+P=0 + P=1 goes below 100. At this point the healths will be scaled up to an "effective" health of
+100%.
+
++-------------+--------------+-----------------+-----------------+
+| P=0 health  | P=1 health   | Traffic to  P=0 |  Traffic to P=1 |
++=============+==============+=================+=================+
+| 100         |  100         | 100%            |   0%            |
++-------------+--------------+-----------------+-----------------+
+| 72          |  72          | 100%            |   0%            |
++-------------+--------------+-----------------+-----------------+
+| 71          |  71          | 99%             |   1%            |
++-------------+--------------+-----------------+-----------------+
+| 50          |  50          | 70%             |   30%           |
++-------------+--------------+-----------------+-----------------+
+| 25          |  100         | 35%             |   65%           |
++-------------+--------------+-----------------+-----------------+
+| 25          |  25          | 50%             |   50%           |
++-------------+--------------+-----------------+-----------------+
+
+As more priorities are added, each level consumes load equal to its "scaled" effective health, so
+P=2 would only receive traffic if the combined health of P=0 + P=1 was less than 100.
+
+
++-------------+-------------+------------+----------------+----------------+----------------+
+| P=0 health  | P=1 health  | P=2 health | Traffic to P=0 | Traffic to P=1 | Traffic to P=3 |
++=============+=============+============+================+================+================+
+| 100         |  100        |  100       | 100%           |   0%           |   0%           |
++-------------+-------------+------------+----------------+----------------+----------------+
+| 72          |  72         |  100       | 100%           |   0%           |   0%           |
++-------------+-------------+------------+----------------+----------------+----------------+
+| 71          |  71         |  100       | 99%            |   1%           |   0%           |
++-------------+-------------+------------+----------------+----------------+----------------+
+| 50          |  50         |  100       | 70%            |   30%          |   0%           |
++-------------+-------------+------------+----------------+----------------+----------------+
+| 25          |  100        |  100       | 35%            |   65%          |   0%           |
++-------------+-------------+------------+----------------+----------------+----------------+
+| 25          |  25         |  100       | 25%            |   25%          |   50%          |
++-------------+-------------+------------+----------------+----------------+----------------+
+
+To sum this up in pseudo algorithms:
+percent load to P=0 == min(100, health(P=0) * 100 / total_health)
```

Comment:
Nit: prefer = to == here, similar to normal expression or function definitions in math.